src/main/java/com/example/springexploring/service/OrderServiceImpl.java

```diff
@@ -83,4 +85,9 @@ private void assertThatUserAreOwner(Long userId, Order order) {
             throw new CustomRuntimeException("User with Id - " + userId + " dont have order with Id - " + order.getId());
     }
 
+    private void assertThatUserAlreadyReceivedOrder(Order order) {
```

Comment:
assertThatOrderIsNotReceived
You don't check that user receives order. It's scenario when we throw exception.